ui/app/components/clients/page/counts.hbs

```diff
@@ -35,52 +34,7 @@
       </Hds::Alert>
     {{/if}}
 
-    {{#if (or @namespace this.namespaces @mountPath this.mountPaths)}}
-      <Hds::Text::Display @tag="p">
```

Comment:
This was these toolbar filters which are removed in the latest designs
<img width="1127" height="252" alt="Screenshot 2025-08-14 at 3 45 22 PM" src="https://github.com/user-attachments/assets/2bd2c55a-7ea4-41f8-88f9-c41bb941063b" />